### ISSUE TITLE
Ability to pass Google Maps API key

### ISSUE THREAD

**Issue by xoolive** [pbugnion/gmaps#61]
I sometimes get the following error (can be stubborn, can fix itself somehow...). No idea why things sometimes work properly without an API key, but I guess it wouldn't hurt to be able to add one?

<img width="1355" alt="capture d ecran 2016-06-23 a 23 44 29" src="https://cloud.githubusercontent.com/assets/1360812/16321343/936d0668-399c-11e6-82c1-c9544db253d1.png">

**Comment by xoolive** [comment]
I don't know whether it is of all a sudden forbidden to get those maps without an API key. 

So I confirm that the page properly works if I add at the top of `jupyter-gmaps.js` the following line:
```javascript
var GoogleMapsLoader = require('google-maps');
GoogleMapsLoader.LIBRARIES = ["visualization"] ;
GoogleMapsLoader.KEY = '>>>> API KEY <<<<';
```

Unfortunately, I am not confident enough with the Python/JS working together to suggest a clean interface... :cry: 

Hoping my workaround may help providing a quick fix!

PS: The API to activate in your Google developer interface is `Google Maps JavaScript API`.  
I followed [these instructions](https://github.com/googlemaps/google-maps-services-python#api-keys).

**Comment by pbugnion** [comment]
Thanks for helping. I've never come across this. Maybe you get rate-limited if you don't specify an API key, and after a certain number or requests, you get banned.

I'm not sure how to pass information from Python to the JS layer outside of the context of widgets, but will look into this.

**Comment by pbugnion** [comment]
For anyone else reading this, a very quick hack for entering your API key is to find the `nbextensions/jupyter-gmaps` directory. On Mac OS X, this is `~/Library/Jupyter/nbextensions/jupyter-gmaps/`. Open the file `index.js` and search for the line that says: `GoogleMapsLoader.LIBRARIES = ["visualization"] ;`

Add a new line below with:

```javascript
GoogleMapsLoader.KEY = "your-api-key"
```

You will need to restart open notebooks.

**Comment by reyale** [comment]
https://developers.google.com/maps/pricing-and-plans/standard-plan-2016-update

"We no longer support keyless access"

**Comment by pbugnion** [comment]
Ah that's very useful, thanks. All map queries now fail unless I set the key first.

**Comment by pbugnion** [comment]
Fixed by PR #63
